### ISSUE TITLE
docker build - googlemock zip file missing

### ISSUE THREAD

**Issue by cmosetick** [metaverse/truss#42]
When executing docker build as noted in DEVELOPING.md one currently gets `unzip: invalid zip magic`. This is because the `gmock-1.7.0.zip` that get downloaded by `autogen.sh` is not an actual zip file / URL. (So curl retrieves a 404 error HTML document instead.)

There seems to have been some rearranging of project locations.
e.g.
https://googlemock.googlecode.com/files/gmock-1.7.0.zip
vs.
https://github.com/google/googletest/tree/master/googlemock

Docker build output:

```
Removing intermediate container 40cd63641b07
Step 9 : RUN ./autogen.sh &&    ./configure --prefix=/usr &&    make &&         make install
 ---> Running in 28558c243895
Google Mock not present.  Fetching gmock-1.7.0 from the web...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1582  100  1582    0     0  11029      0 --:--:-- --:--:-- --:--:-- 12359
unzip: invalid zip magic 4F44213C
The command '/bin/sh -c ./autogen.sh &&         ./configure --prefix=/usr &&    make &&         make install' returned a non-zero code: 1
```

Note that autogen.sh has already been fixed on the master branch of the new location for googlemock:
https://raw.githubusercontent.com/google/protobuf/master/autogen.sh

However, the Dockerfile is currently pointing to branch `3.0.0-beta-4`.

**Comment by hasIan** [comment]
Hi Chris, thanks for the report. I'll be pushing an update soon that takes advantage of the prebuilt binaries for the now-released protobuf 3.0.0 implementation that should alleviate this issue.

**Comment by cmosetick** [comment]
Thanks @hasIan I've also submitted PR #43 for you to review that fixes this issue and cleans up the current Dockerfile.
